package.json

```diff
@@ -1,114 +1,21 @@
-    "build-module": "npm run clean-module && tsc -p tsconfig.json",
-    "build-browser": "npm run clean-browser && webpack --config webpack.browser.js --env.env=prod",
-    "analyze-bundle": "npm run clean-browser && webpack --config webpack.config.js --env.env=analyze",
-    "lint": "eslint \"src/**/*.ts\" && eslint \"__tests__/**/*.ts\"",
-    "lint-fix": "eslint --config .eslintrc.json --fix \"src/**\" && eslint --config .eslintrc.json --fix \"__tests__/**/*.ts\"",
-    "format": "prettier --write \"src/**\" \"__tests__/**\"",
-    "licenses": "license-checker --production --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
-    "dev": "tsc-watch --preserveWatchOutput --noClear",
-    "dev-server": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/server && npm run dev\"",
-    "dev-script": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/script && npm run dev\"",
-    "dev-bundle": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/bundle && npm run dev\"",
-    "dev-ios": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/native && npm run dev-ios\"",
-    "dev-android": "concurrently \"npm run dev\" \"npm run dev-test-helper\" \"cd examples/native && npm run dev-android\"",
-    "dev-test-helper": "cd examples/test-helper && npm run dev",
     "test": "npm run lint && npm run licenses && npm run test-unit",
-    "test-unit": "jest --coverage --verbose",
-    "test-acceptance": "npm run test-acceptance-server && npm run test-acceptance-script && npm run test-acceptance-bundle && npm run test-acceptance-ios && npm run test-acceptance-android",
-    "test-acceptance-server": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-script": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-bundle": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-ios": "echo \"Not Implemented\" && exit 1",
-    "test-acceptance-android": "echo \"Not Implemented\" && exit 1",
-    "bootstrap-examples": "./bootstrap-examples.bash",
-    "build-api-docs": "npx typedoc --out docs/api/source/api",
-    "build-docs-preview-site": "npm run build-api-docs; cd docs/prose; make html; cd ../api; make html; cd ../; rm -r dist || true; mkdir -p dist/prose dist/api; cp -r prose/build/html/. dist/prose/; cp -r api/build/html/. dist/api/; echo 'Draft documentation: <a href=\"./prose/\">Prose</a>, <a href=\"./api/\">API docs</a>.' >> dist/index.html"
+    "test-unit": "jest --coverage --verbose"
```

Comment:
+1 on the `postinstall`. It seems to be mentioned as a good idea, although I only found a couple of references.
As for adding more script options - do you mean just being able to type `npm build` from the root directory as a convenience over having to type `lerna run build` (i.e. saving 6 keystrokes), or something more than that?

Reply:
Yes - because `lerna run build` expects `lerna` to be available on your `$PATH` (which is recommended against), and `npx lerna run build` obscures it if Lerna does not happen to be added as a dependency, potentially leading to people using different versions and "it works on my machine" bugs. And most of all: when I open a new repository, I usually look at the `scripts` in `package.json` to determine what commands should be run, whereas you'd have to remember that `lerna run build` is the correct invocation. And it's just nice if all our projects (and npm projects generally) just consistently work with `npm test` and `npm run build`.

Reply:
Cool - thanks for that elaboration @Vinnl. Seems to be a good argument for `lerna init` adding a bunch of 'standard' script options for all new setups by default! Anyway, I'll add the basic build and test ones, and use `.node_modules/.bin/lerna build` to use the local Lerna (and not assume a global install).

Reply:
@NSeydoux Right, so the addition could just be:

```
  "build": "lerna run build"
```

to the top-level package.json.

@pmcb55 Great. Note that you don't need to explicitly refer to `./node_modules/.bin/lerna` - when npm runs scripts from the `scripts` property in package.json, it automatically adds binaries from the project's dependencies to the `$PATH` for that script.